keps/sig-scheduling/902-non-preempting-priorityclass/README.md

```diff
@@ -199,18 +205,126 @@ Ensure existing tests (for preempting PriorityClasses) do not break.
+
+<!--
+This section must be completed when targeting beta to a release.
+-->
+
+###### How can an operator determine if the feature is in use by workloads?
+The operator can determine if the workload is using the feature by checking if the priorityclass's preemptionPolicy is set to "Never".
+<!--
+Ideally, this should be a metric. Operations against the Kubernetes API (e.g.,
+checking if there are objects with field X set) may be a last resort. Avoid
+logs or events for this purpose.
+-->
+
+###### How can someone using this feature know that it is working for their instance?
+
+<!--
+For instance, if this is a pod-related feature, it should be possible to determine if the feature is functioning properly
+for each individual pod.
+Pick one more of these and delete the rest.
+Please describe all items visible to end users below with sufficient detail so that they can verify correct enablement
+and operation of this feature.
+Recall that end users cannot usually observe component logs or access metrics.
+-->
+
+- [x] Events
+  - Event Reason: There is an event sent by kube-scheduler if the pod preempts other pods. If the feature is working and the pod with the priorityclass'preemptionPolicy as Never, there won't be a preemption related event for this pod.
+- [ ] API .status
+  - Condition name:
+  - Other field:
+- [x] Other (treat as last resort)
+  - Details: Check if pods with preemptionPolicy set to Never can preempt other low-priority pods when the cluster resources cannot be met.  
+
+###### What are the reasonable SLOs (Service Level Objectives) for the enhancement?
+N/A
+
+<!--
+This is your opportunity to define what "normal" quality of service looks like
+for a feature.
+
+It's impossible to provide comprehensive guidance, but at the very
+high level (needs more precise definitions) those may be things like:
+  - per-day percentage of API calls finishing with 5XX errors <= 1%
+  - 99% percentile over day of absolute value from (job creation time minus expected
+    job creation time) for cron job <= 10%
+  - 99.9% of /health requests per day finish with 200 code
+
+These goals will help you determine what you need to measure (SLIs) in the next
+question.
+-->
+
+###### What are the SLIs (Service Level Indicators) an operator can use to determine the health of the service?
+
+<!--
+Pick one more of these and delete the rest.
+-->
+
+- [x] Metrics
+  - Metric name: preemption_victims
+  - [Optional] Aggregation method:
+  - Components exposing the metric: kube-scheduler
+- [ ] Other (treat as last resort)
+  - Details:
+
+###### Are there any missing metrics that would be useful to have to improve observability of this feature? 
+We currently only have events that describe a pod being preempted by another pod. But we don't
+have an event that describes why sometimes the preemption is not successful. We can enhance the
+message of the existing event for scheduling failed to include details about preemption. This
```

Comment:
Add this to graduation criteria.

Also, avoid amending commits so that it's easier to visualize the evolution of the doc.

You can squash once the PR is approved.

Reply:
Done